### PR TITLE
E2e weekly

### PR DESCRIPTION
**Pull request by Mashyuf**
Added e2e file for both weekly and monthly view.

### CODE REVIEW

**Review by michellelyHuang728 — COMMENTED** [review]
good

**Review by michellelyHuang728 — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
       "license": "GPL-3.0-only",
       "dependencies": {
         "electron-squirrel-startup": "^1.0.1",
-        "node": "^18.20.3"
+        "node": "^18.20.3",
+        "wdio": "^6.0.1"
```
Comment (katulevskiy) [inline package-lock.json:14]: why did you add wdio as dependency? should be only for dev testing, not building the application for final user

File: source/__tests__/e2e/test.e2e.js [hunk 2]
```diff
@@ -1,8 +1,86 @@
...
+    (await $("#prev-button")).click();
+    (await $("#cur-button")).click();
+    const currentweekDate = (await (await $(".day > header")).getText()).slice(-2);
+    await expect(todayDate - currentweekDate).toBe(0);
+  });
+
+  //  it("check adding a new task with time", async() => {
+  //   (await $("#add-button")).click();
+  //   (await $("#new-task-title")).setValue("New Meeting");
+  //   (await $("#new-task-description")).setValue("New meeting with colleagues");
```
Comment (katulevskiy) [inline source/__tests__/e2e/test.e2e.js:75]: remove comented out code

File: source/__tests__/tmp.test.js
Comment (katulevskiy) [inline source/__tests__/tmp.test.js:1]: remove this. what is temporary file doing here?

File: source/__tests__/tmp2.js
Comment (katulevskiy) [inline source/__tests__/tmp2.js:1]: temp file

File: source/__tests__/tmp3.js
Comment (katulevskiy) [inline source/__tests__/tmp3.js:1]: temp file
